### DISCUSSION TITLE
IOS Snackbar like behaviour

### DISCUSSION THREAD

**Discussion by Sanyos** [NativeScript/NativeScript#10263] [Q&A] [answered]
Hi guys!

I made a custom snackbar component, that only displays some success or warning or error response texts with custom styling and design according to my app. It auto closes itself after a few seconds.
Basically it only displays an element with 90% width and 50 height in an absolute layout positioned to the bottom of the screen. It works like charm on android, but on ios when the snackbar is displayed i can't scroll the page under it or even i can not tap or interact with any other elements till the snackbar disappears. It seems like it creates a full screen transparent background layer with the snackbar. 
Again the is not occuring on android, only on ios. 

Any idea how to work around it?

This is how i display the snackbar in my app: 

`<AbsoluteLayout [visibility]="(showSnackbar$ | async) ? 'visible' : 'collapse'">
    <FlexboxLayout justifyContent="center" width="100%" >
      <snackbar></snackbar> 
    </FlexboxLayout>
 </AbsoluteLayout>`

**Comment by rigor789** [comment, marked as answer]
You can try setting `isUserInteractionEnabled="false"` on the `AbsoluteLayout`.

_ps.: For positioning though, I would recommend using a `GridLayout` and `rows="*, auto"` and placing the snackbar into `row="1"` to push it to the bottom of the screen. It's usually simpler than positioning with AbsoluteLayout._

**Reply by Sanyos** [reply]
Thanks, works like charm.
